### PR TITLE
Fix Watchers and Linking Issue

### PR DESCRIPTION
**Pull request by maira-samtek**

### CODE REVIEW

File: src/macfc-security-hub-sync.ts [hunk 1]
```diff
@@ -446,7 +446,7 @@ export class SecurityHubJiraSync {
 
     // Validate the extracted region
     if (!isAwsRegion(region)) {
-      console.error(`Invalid AWS region: ${region}`)
+      //console.error(`Invalid AWS region: ${region}`)
```
Comment (yonassrobi) [inline src/macfc-security-hub-sync.ts:449]: Any reason why this line needs to be commented out?

**Review by yonassrobi — APPROVED** [review]
LGTM. Added one question.

File: src/macfc-security-hub-sync.ts (hunk 1 above)
Comment (maira-samtek) [reply src/macfc-security-hub-sync.ts:449]: The logs looked so ugly and were used to indicate the finding url type where the finding id does not contain the region. However, our logic sets the region to us-east-1 as fall-back logic so to keep the logs in run as short as possible I removed this extra log
image

Comment (maira-samtek) [reply src/macfc-security-hub-sync.ts:449]: cleaner logs helps w debugging when needed

**Pull request merged**